src/styles/blocks/main/cards-container.scss

```diff
@@ -0,0 +1,24 @@
+.cards-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  width: $cards-container-width;
+  padding: {
+    top: 50px;
+    bottom: 72px;
```

Comment:
Generally try to stick to values divisible by 5, but that's just me nitpicking :P

Reply:
Okay, but test's force me to do it pixel perfect.

Reply:
General rule of thumb is to create something that's easy to work with and fairly represents the design. If the designer requires you such values and it's not up for a debate (explaining to him that it will be better doesn't work) then definitely stick to such values.